include/MMD/aocl_mmd.h

```diff
@@ -484,7 +484,13 @@ typedef enum {
    *  memory bank. It is invalid to specify this property without also specifying
    *  AOCL_MMD_GLOBAL_MEMORY_INTERFACE.
    */
-  AOCL_MMD_MEM_PROPERTIES_MEMORY_BANK
+  AOCL_MMD_MEM_PROPERTIES_MEMORY_BANK,
+  /**
+   *  Specifies the buffer location used for USM API calls.  The buffer location
+   *  is an integer index that corresponds to a global memory location in the
+   *  board_spec.xml file.  Similar to AOCL_MMD_MEM_PROPERTIES_GLOBAL_MEMORY.
+   */
+  AOCL_MMD_MEM_PROPERTIES_BUFFER_LOCATION
```

Comment:
I believe C++11 allows trailing comma now, which is nice for minimizing changes and conflicts :slightly_smiling_face: 
```suggestion
  AOCL_MMD_MEM_PROPERTIES_BUFFER_LOCATION,
```

Reply:
Oh really!?  That is fantastic!  Is there a guarantee that users of this header will be compiling with C++11 or greater?